components/omnibox/browser/commander_provider_unittest.cc

```diff
@@ -194,7 +194,7 @@ TEST_F(CommanderProviderTest, RemovingPrefixClearsMatches) {
   EXPECT_EQ(0u, provider()->matches().size());
 }
 
-TEST_F(CommanderProviderTest, PromptingForMoreInputSetsAnnotation) {
+TEST_F(CommanderProviderTest, PromptingForMoreInputSetsNoAnnotation) {
```

Comment:
Might as well just remove this test.

Reply:
I thought as much, but didn't want to do it without checking with you first. I'll remove it then.